setup.py

```diff
@@ -446,6 +446,7 @@ def do_setup():
         # DEPENDENCIES_EPOCH_NUMBER in the Dockerfile
         #####################################################################################################
         install_requires=[
+            'ConfigUpdater>=1.0.1',
```

Comment:
Let's maintain the alphabetical order.

Reply:
@kaxil it's maintained... "C" is before "a".

Reply:
Hahaha.. because of the capital letter.. oh goddd